.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Added a `.gitignore` file for you. You don't need to upload node_modules to git.